src/amo/components/SearchForm/index.js

```diff
@@ -27,6 +27,7 @@ import Icon from 'ui/components/Icon';
 
 import './styles.scss';
 
+export const SEARCH_TERM_MIN_LENGTH = 2;
```

Comment:
This should be set to `3` because as-is only a `1` character search is too short, but I think we wanted a three character minimum.